src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -210,9 +210,9 @@ function AppSettingsModal({
                   </div>
                 )}
               />
-              {formikProps.values.enabled && children
+              {(formikProps.values.enabled || configureBeforeEnable) && children
               && <AppConfigFormDivider marginAdj={{ default: 0, sm: 0 }} />}
-              <AppSettingsForm formikProps={formikProps}>
+              <AppSettingsForm formikProps={formikProps} showForm={formikProps.values.enabled || configureBeforeEnable}>
```

Comment:
spacing issue in footer.

current design:
<img width="841" alt="Screenshot 2021-09-10 at 2 55 20 PM" src="https://user-images.githubusercontent.com/79941147/132855249-b05437cd-1b23-4f2e-bd88-000d281a6db2.png">

Figma Design:
<img width="741" alt="Screenshot 2021-09-10 at 5 45 52 PM" src="https://user-images.githubusercontent.com/79941147/132855293-75b83261-96b2-46cc-bccf-4af848c651b6.png">

Reply:
Reopening this. You do not need to fix this issue in paragon. These changes can be done in this MFE.

Reply:
I'm not saying that they can't be done using CSS. Just that I was told to avoid customising the UI as much as possible so that all modals, etc look the same across the platform. If I am using a paragon component in two places, it should look the same, which won't happen if we layer styles on top of it.